semcore/date-picker/src/DateRangePicker.jsx

```diff
@@ -32,10 +32,10 @@ class DateRangePickerRoot extends RangePickerAbstract {
   navigateStep = 'month';
   keyStep = 'day';
   keyDiff = {
-    37: -1,
-    38: -7,
-    39: 1,
-    40: 7,
+    ArrowLeft: -1,
```

Comment:
It should have been done a long time ago, but looks like you forgot about adding a same change to `DatePickerAbstract`